politeiawww/client/records.go

```diff
@@ -75,6 +76,24 @@ func (c *Client) RecordSetStatus(ss rcv1.SetStatus) (*rcv1.SetStatusReply, error
 	return &ssr, nil
 }
 
+// RecordSetBillingStatus sends a records v1 SetBillingStatus request
+// to politeiawww.
+func (c *Client) RecordSetBillingStatus(sbs piv1.SetBillingStatus) (*piv1.SetBillingStatusReply, error) {
```

Comment:
The `records.go` file is for the `records` API.  This function should be moved into a `pi.go` file and the name should be updated to be `PiSetBillingStatus` to reflect that it is a `pi` API call.

Reply:
💯